tests/test_models/test_engine/test_file_storage.py

```diff
@@ -114,6 +114,42 @@ def test_save(self):
             js = f.read()
         self.assertEqual(json.loads(string), json.loads(js))
 
+    @unittest.skipIf(models.storage_t == 'db', "not testing db storage")
```

Comment:
I suggest you have some symmetry in your skip decorator arguments.
Right now you have:
@unittest.skipIf(models.storage_t != 'db', "testing db storage")
@unittest.skipIf(models.storage_t == 'db', "not testing db storage")

I suggest:
@unittest.skipIf(models.storage_t != 'db', "testing db storage")
@unittest.skipIf(models.storage_t == 'db', "testing file storage")

Or:
@unittest.skipIf(models.storage_t != 'db', "not testing file storage")
@unittest.skipIf(models.storage_t == 'db', "not testing db storage")